### PR TITLE
Updated CONTRIBUTING.md fixes #942

### LINKED ISSUE #942: Update references on CONTRIBUTING.md to slack
## Description

As a contributor,
I need updated references to communication channels used by the community,
so that I can reach out to the community in the correct channels.

Contributing guidelines has still a reference to Slack channels. It should have a link to Zulip and the appropriate stream. This should minimize confusion from newcomers

### Update [Required]
- [ ] Update CONTRIBUTING.md to have links to Zulip space and stream instead of Slack channel.

## Definition of Done
- [ ] All of the required items are completed.
- [x] Approval by 1 mentor.

## Estimation

30 min

### PR DESCRIPTION
**Pull request by Harirai**
Updated Reference links.

# Description
Updated broken/old links.

Fixes #942

# Type of Change:
- User Interface
- Outreach
- Documentation

# How Has This Been Tested?
By opening updated links.

# Checklist:

- [x] I have made corresponding changes to the documentation

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Contributing Guidelines
-* You can join our [slack channel](http://systers.io/slack-systers-opensource/).  Each active repo has its own channel to direct questions to (for example #powerup or #portal).  
+* You can join our [ AnitaB.org Open Source Zulip Channel](https://anitab-org.zulipchat.com/).  Each active repo has its own channel to direct questions to (for example [#powerup](https://anitab-org.zulipchat.com/#narrow/stream/222537-powerup) or [#portal](https://anitab-org.zulipchat.com/#narrow/stream/222540-portal)).
```
Comment (isabelcosta) [inline CONTRIBUTING.md:2]: Can you please add here somewhere the specific stream dedicated to #vms?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # Contributing Guidelines
-* You can join our [slack channel](http://systers.io/slack-systers-opensource/).  Each active repo has its own channel to direct questions to (for example #powerup or #portal).  
+* You can join our [ AnitaB.org Open Source Zulip Channel](https://anitab-org.zulipchat.com/).  Each active repo has its own channel to direct questions to (for example [#powerup](https://anitab-org.zulipchat.com/#narrow/stream/222537-powerup) or [#portal](https://anitab-org.zulipchat.com/#narrow/stream/222540-portal)).  
 * Remember that this is an inclusive community, committed to creating a safe, positive environment.  See the full [Code of Conduct](http://www.systers.io/code-of-conduct.html).
 ## General Guidelines
-* If you’re just getting started work on an issue labeled “First Timers Only” in any project. Additional resources are available on our [website](http://www.systers.io).
```
Comment (isabelcosta) [inline CONTRIBUTING.md:5]: systers.io is still active http://systers.io/
anitab.org does not have any information about open source. We will eventually build a open source website to replace systers.io, but that is future

File: CONTRIBUTING.md (hunk 1 above)
Comment (Harirai) [reply CONTRIBUTING.md:2]: Okay

File: CONTRIBUTING.md (hunk 2 above)
Comment (mayburgos) [reply CONTRIBUTING.md:5]: I think removing it for now until we build out the open source one is better.

Comment (Harirai) [reply CONTRIBUTING.md:5]: @isabelcosta  So, should I remove this link, as May suggested. Or let it link to systers.io ?

Comment (isabelcosta) [reply CONTRIBUTING.md:5]: do remove it @Harirai . In this case @mayburgos knows best ;)

Comment (Harirai) [reply CONTRIBUTING.md:5]: @isabelcosta  Okay I'll remove it.

**Comment by Harirai** [comment]
A new PR was made to fix the same issue ([#964](https://github.com/anitab-org/vms/pull/964)), so closing this PR.

**Pull request closed without merging**